### PR TITLE
Web app integration test

### PR DESCRIPTION
**Pull request by songkaiape**
Add WebApp integration test cases
1. testNewWebApp
2. testNewWebApppWithExistedASP
3. testNewWebAppWithExistedRgAndASP
4. testNewWebAppWithExistedRg
5. testNewWebAppWithJar
6. testExistedWebApp

### CODE REVIEW

**Comment by vscjavaci** [comment]
Can one of the admins verify this patch?

File: PluginsAndFeatures/azure-toolkit-for-intellij/Test/java/com/microsoft/intellij/tooling/TestSchedulerProvider.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.microsoft.intellij.tooling;
```
Comment (jdneo) [inline PluginsAndFeatures/azure-toolkit-for-intellij/Test/java/com/microsoft/intellij/tooling/TestSchedulerProvider.java]: add license

File: PluginsAndFeatures/azure-toolkit-for-intellij/Test/java/com/microsoft/intellij/runner/webapp/WebAppIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,318 @@
+package com.microsoft.intellij.runner.webapp;
```
Comment (jdneo) [inline PluginsAndFeatures/azure-toolkit-for-intellij/Test/java/com/microsoft/intellij/runner/webapp/WebAppIntegrationTest.java]: add license

Comment (songkaiape) [reply PluginsAndFeatures/azure-toolkit-for-intellij/Test/java/com/microsoft/intellij/runner/webapp/WebAppIntegrationTest.java]: Got it

File: PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle [hunk 3]
```diff
@@ -95,8 +96,22 @@ dependencies {
     testCompile 'info.cukes:cucumber-java:1.2.5'
     testCompile 'org.mockito:mockito-core:2.7.22'
     testCompile 'org.assertj:assertj-swing-junit:3.5.0'
+
+    testCompile 'com.github.tomakehurst:wiremock:2.8.0'
+    testCompile 'org.powermock:powermock-module-junit4:1.7.0RC4'
+    testCompile 'org.powermock:powermock-api-mockito2:1.7.0RC4'
+    compile ('javax.servlet:javax.servlet-api:3.1.0'){
```
Comment (jdneo) [inline PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle]: try to use ```testCompile``` and remove ```force=true```?

**Pull request merged**